cmd/sealer/cmd/cluster/run.go

```diff
@@ -510,3 +518,16 @@ func prepareMaterials(infraDriver infradriver.InfraDriver, imageEngine imageengi
 	}
 	return nil
 }
+
+func checkClusterIsExists(cf clusterfile.Interface) (string, error) {
+	rawCluster := cf.GetCluster()
+	clusterFile, _, err := clusterfile.GetActualClusterFile()
+	if err == nil {
+		cluster := clusterFile.GetCluster()
+		if cluster.Spec.Image != rawCluster.Spec.Image {
+			return "exist", fmt.Errorf("the cluster image already exists, please uninstall the current cluster and install another version of the cluster image")
```

Comment:
"exist" is not good, you should use const such as "SameImageName" Or something to identity it